pkg/ccl/importccl/import_stmt_test.go

```diff
@@ -1305,13 +1305,13 @@ func TestImportMysql(t *testing.T) {
 	}{
 		{`read data only`, expectSimple, `IMPORT TABLE simple (i INT PRIMARY KEY, s text, b bytea) MYSQLDUMP DATA ($1)`, simple},
 		{`single table dump`, expectSimple, `IMPORT TABLE simple FROM MYSQLDUMP ($1)`, simple},
-		{`single table dump`, expectSecond, `IMPORT TABLE second FROM MYSQLDUMP ($1)`, second},
+		{`second table dump`, expectSecond, `IMPORT TABLE second FROM MYSQLDUMP ($1)`, second},
 		{`simple from multi`, expectSimple, `IMPORT TABLE simple FROM MYSQLDUMP ($1)`, multitable},
 		{`second from multi`, expectSecond, `IMPORT TABLE second FROM MYSQLDUMP ($1)`, multitable},
 		{`all from multi`, expectAll, `IMPORT MYSQLDUMP ($1)`, multitable},
 	} {
 		t.Run(c.name, func(t *testing.T) {
-			sqlDB.Exec(t, `DROP TABLE IF EXISTS simple, second`)
+			sqlDB.Exec(t, `DROP TABLE IF EXISTS simple, second, everything`)
```

Comment:
`everything` is one of the 3 tables in the `multitable` dump (`db.sql`): https://github.com/cockroachdb/cockroach/blob/master/pkg/ccl/importccl/testdata/mysqldump/db.sql#L25

It doesn't have any rows (yet), since it was mostly added to check the various column type conversions.

Reply:
nevermind i see now